### DISCUSSION TITLE
Mapbox Pricing

### DISCUSSION THREAD

**Discussion by codscino** [maplibre/flutter-maplibre-gl#176] [Q&A]
Hi, I would like to know if I use maplibre on my flutter app on IOS and android will I be able to be billed by Mapbox by MAU (monthly active users) and not by tile requests, like in the official mapbox sdk for IOS and android.

**Comment by mariusvn** [comment]
if you use maplibre, you don't need to use the MapBox tile servers. If you do, be sure you are in your rights by not using it with mapbox-gl.

**Comment by mariusvn** [comment]
Btw I think the mapbox style protocol (mapbox://) was removed from Maplibre so idk if you'll be able to use the mapbox style
